libs/domains/variables/feature/src/lib/output-variables/output-variables.tsx

```diff
@@ -0,0 +1,192 @@
+import {
+  type SortingState,
+  createColumnHelper,
+  flexRender,
+  getCoreRowModel,
+  getExpandedRowModel,
+  getSortedRowModel,
+  useReactTable,
+} from '@tanstack/react-table'
+import { type VariableResponse as Variable } from 'qovery-typescript-axios'
+import { type ComponentProps, Fragment, useMemo, useState } from 'react'
+import { match } from 'ts-pattern'
+import { Icon, IconAwesomeEnum, TablePrimitives, Tooltip } from '@qovery/shared/ui'
+import { useCopyToClipboard } from '@qovery/shared/util-hooks'
+import { twMerge } from '@qovery/shared/util-js'
+import { useVariables } from '../hooks/use-variables/use-variables'
+
+const { Table } = TablePrimitives
+
+function VariableValue({ variable }: { variable: Variable }) {
+  const [visible, setVisible] = useState(false)
+  const [icon, setIcon] = useState(IconAwesomeEnum.COPY)
+  const [, copyToClipboard] = useCopyToClipboard()
+
+  const onClickCopyToClipboard = (content: string) => {
+    copyToClipboard(content)
+    setIcon(IconAwesomeEnum.CHECK)
+    setTimeout(() => {
+      setIcon(IconAwesomeEnum.COPY)
+    }, 1000)
+  }
+
+  return variable.is_secret ? (
+    <span className="flex items-center gap-2 text-sm text-neutral-300">
+      <Tooltip content="Secret variable">
+        <span>
+          <Icon className="block w-4" name={IconAwesomeEnum.LOCK} />
+        </span>
+      </Tooltip>
+      <span className="text-2xl font-medium pt-1.5">*************</span>
+    </span>
+  ) : (
+    <span className="flex items-center gap-2 text-sm">
+      <Tooltip content={visible ? 'Hide variable' : 'View variable'}>
+        <button type="button" className="w-4 text-brand-500" onClick={() => setVisible((visible) => !visible)}>
+          {visible ? <Icon name={IconAwesomeEnum.EYE_SLASH} /> : <Icon name={IconAwesomeEnum.EYE} />}
+        </button>
+      </Tooltip>
+      {visible ? (
+        <>
+          <span className="text-brand-500">{variable.value}</span>
+          {Boolean(variable.value) && (
+            <Tooltip content="Copy">
+              <button
```

Comment:
You can use `CopyToClipboardButtonIcon` component

Reply:
Fixed https://github.com/Qovery/console/pull/1177/commits/2cd646cff22337ba2e91411a08b943e5aece4feb